priv/repo/migrations/dummy_migration.ex

```diff
@@ -0,0 +1,2 @@
+# Travis CI does not include empty folder
+# This file is here so that Travis CI can recognize this folder
```

Comment:
If you have no migrations, why does your README tell you to run `mix ecto.migrate`? Can you get away with just `mix ecto.create`?

Reply:
Boo. That stinks. Ok, fair enough

Reply:
Sorry. I think I was not clear on this. 
The README was auto generated by Phoenix. I didn't write it myself. 
I didn't have to do `mix ecto.migrate`
But I meant, I tried to include `mix ecto.create` in the travis.yml file but it didn't work. I had to insert a dummy empty migration file in this folder for the build to pass.